frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -253,12 +253,23 @@ export const StyledLogoLink = styled.a(({}) => ({
   },
 }))
 
-export const StyledLogo = styled.img(({ theme }) => ({
-  height: "1.5rem",
-  maxWidth: "15rem",
-  margin: "0.25rem 0.5rem 0.25rem 0",
-  zIndex: theme.zIndices.header,
-}))
+export interface StyledLogoProps {
+  sidebarWidth?: string
+}
+
+export const StyledLogo = styled.img<StyledLogoProps>(
+  ({ theme, sidebarWidth }) => ({
+    height: "1.5rem",
+    margin: "0.25rem 0.5rem 0.25rem 0",
```

Comment:
As part of the advanced theming preparations, I started to replace hardcoded theming variables with variables from our theme. For example in this case: 

```python
  marginTop: theme.spacing.twoXS,
  marginBottom: theme.spacing.twoXS,
  marginLeft: theme.spacing.none,
  marginRight: theme.spacing.sm,
```

For `height`, we could either use: `theme.iconSizes.xl` or maybe add a new variable `logoHeight` to sizes.